style.css

```
@@ -39,3 +81,50 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+.inputNewTasks{
+  height: 50px;
+  padding: 8px;
+  flex-direction: row;
+  align-items: stretch;
+  display:flex;
+  font-family: 'NEXON Lv1 Gothic OTF';
+  margin-top: auto;
+  box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
+
+}
+
+
+
+#todoInput{
+  flex: 1; /*specify it for a child if you wnat it to fill the remaining horizontal spaces*/
+  border-radius: 10px;
+  padding: 0.2px;
+  margin-right: 5px;
+  text-align: center;
+  font-size: 1em;
+
```

```suggestion
border: 1px solid rgba(0, 0, 0, 0.1);
```
로 인풋의 border를 스타일링 해주시면 좋을 것 같아요~ [참고](https://developer.mozilla.org/ko/docs/Web/CSS/border)